src/components/Header/web3/NetworkModal/Networks.tsx

```diff
@@ -197,6 +222,16 @@ const Networks = ({
                     {name}
                   </Text>
                 </Flex>
+                {state === ChainState.NEW && (
```

Comment:
```suggestion
                {isNew && (
```

Reply:
I often create a new variable only when using it >= 2 times :D